clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -666,6 +667,9 @@ mlir::cir::GlobalOp CIRGenModule::createGlobalOp(CIRGenModule &CGM,
     // since MLIR doesn't allow public declarations.
     mlir::SymbolTable::setSymbolVisibility(
         g, mlir::SymbolTable::Visibility::Private);
+
+    g.setVisibilityAttrAttr(
```

Comment:
Since this is a mandatory attribute, we should incorporate default creation in the builders for funcop and globalop, so that we only need to call it explicitly whenever we have more meaningful setup to do.

Reply:
These calls also haven't been resolved (now that you create the default ones in the builders)